### PR TITLE
Fix bug on setting plotly figure as State.vars

### PR DESCRIPTION
**Pull request by lawrence-axb**
### All Submissions:

- [x] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [x] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### New Feature Submission:

- [x] Does your submission pass the tests? 
- [x] Have you linted your code locally prior to submission?

### Changes To Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

### **After** these steps, you're ready to open a pull request.

    a. Give a descriptive title to your PR.
Fix bug on setting plotly figure as State.vars

    b. Describe your changes.
plotly.figure was not accepted even the TypeError said that Plotly figures is acceptable

    c. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

**Review by picklelo — APPROVED** [review]
Sweet thanks for this

**Review by picklelo — CHANGES_REQUESTED** [review]
Looks like this needs to be ilnted with `black`

**Comment by lawrence-axb** [comment]
@picklelo Thanks! I missed it :D

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -93,6 +93,6 @@ find pynecone tests -name "*.py" -not -path pynecone/pc.py | xargs poetry run da
 ```
 Finally run `black` to format your code.
 ``` bash
-poetry run black pynecone
+poetry run black pynecone tests
```
Comment (picklelo) [inline CONTRIBUTING.md:96]: thanks for adding!

**Review by picklelo — APPROVED** [review]

**Pull request merged**